content/servicemesh/download.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Download and Install Istio CLI"
+date: 2018-11-13T16:36:43+09:00
+weight: 20
+draft: true
+---
+
+Before we can get started configuring Istio we’ll need to first install the command line tools that you will interact with. To do this run the following.
+
+```
+cd ~/environment
+
+curl -L https://git.io/getLatestIstio | sh -
+
+cd istio-1.0.x
```

Comment:
this command failed for me... let's use a wildcard instead?:
`cd istio-1.*`

Reply:
it was to make sure istio version is keep upgrading.. I fixed it to current 1.0.3